### PR TITLE
Remove debug noise from stdout and tidy errors to stderr

### PR DESCRIPTION
**Pull request by nedgladstone-rally**
Remove runtime debug noise from saml-utils
Retained one startup-time message, because stdout/stderr messages from saml-utils at zuul startup time are logged to Splunk.

### CODE REVIEW

File: src/main/java/com/rallydev/saml/SAMLResponseValidator.java [hunk 1]
```diff
@@ -192,12 +188,9 @@ private void validateSignature(Signature signature, String issuer) throws Valida
             throw new ValidationException("the signature was null");
         }
         Credential credential = credentials.get(issuer);
-        System.out.println("NG: SAMLResponseValidator.validateSignature - trusted issuers: " + getTrustedIssuers());
         if (credential == null) {
-            System.out.println("NG: SAMLResponseValidator.validateSignature - invalid issuer " + issuer);
             throwNotTrustedIssuerException(issuer);
```
Comment (kworam-rally) [inline src/main/java/com/rallydev/saml/SAMLResponseValidator.java:192]: should this be System.err.println like above?

File: src/main/java/com/rallydev/saml/SAMLResponseValidator.java [hunk 2]
```diff
@@ -227,12 +220,9 @@ private void validateAssertion(Response samlResponse) throws ValidationException
         }
         Assertion assertion = samlResponse.getAssertions().get(0);
         String issuer = assertion.getIssuer().getValue();
-        System.out.println("NG: SAMLResponseValidator.validateAssertion - trusted issuers: " + getTrustedIssuers());
         if (!credentials.containsKey(issuer)) {
-            System.out.println("NG: SAMLResponseValidator.validateAssertion - invalid issuer " + issuer);
             throwNotTrustedIssuerException(issuer);
```
Comment (kworam-rally) [inline src/main/java/com/rallydev/saml/SAMLResponseValidator.java:224]: same comment as above

File: src/main/java/com/rallydev/saml/SAMLResponseValidator.java [hunk 3]
```diff
@@ -242,12 +232,9 @@ private void throwNotTrustedIssuerException(String issuer) throws ValidationExce
     private void validateCredentials(Response samlResponse) throws ValidationException {
         new ResponseSchemaValidator().validate(samlResponse);
         String issuer = samlResponse.getIssuer().getValue();
-        System.out.println("NG: SAMLResponseValidator.validateCredentials - trusted issuers: " + getTrustedIssuers());
         if (!credentials.containsKey(issuer)) {
-            System.out.println("NG: SAMLResponseValidator.validateCredentials - invalid issuer " + issuer);
             throwNotTrustedIssuerException(issuer);
         }
```
Comment (kworam-rally) [inline src/main/java/com/rallydev/saml/SAMLResponseValidator.java:237]: same comment as above

File: src/main/java/com/rallydev/saml/SAMLResponseValidator.java (hunk 1 above)
Comment (nedgladstone-rally) [reply src/main/java/com/rallydev/saml/SAMLResponseValidator.java:192]: I removed all of the runtime outputs and only kept the startup-time ones.

**Review by kworam-rally — APPROVED** [review]

**Pull request merged**